ourrevolution/settings.py

```diff
@@ -51,6 +64,14 @@
     'EVENTS_PROMOTE_MAX',
     4000
 ))
+EVENTS_PROMOTE_MAX_DISTANCE_MILES = int(os.environ.get(
+    'EVENTS_PROMOTE_MAX_DISTANCE_MILES',
+    100
```

Comment:
We should probably set this config to something much smaller, but this default is fine.

Reply:
seems ok to me? its going to sort by distance anyway if theres too many.

Reply:
I don't see many scenarios where people will be driving 100 miles to an event, and sending them an email for an event they'd never attend only risks an unsubscribe. We can talk about an appropriate number.